dist/js/i18n/si.js

```diff
@@ -0,0 +1,3 @@
+/*! Select2 4.0.3 | https://github.com/select2/select2/blob/master/LICENSE.md */
+
+(function(){if(jQuery&&jQuery.fn&&jQuery.fn.select2&&jQuery.fn.select2.amd)var e=jQuery.fn.select2.amd;return e.define("select2/i18n/si",[],function(){return{errorLoading:function(){return"Napaka pri nalaganju rezultatov"},inputTooLong:function(e){var t=e.input.length-e.maximum,n="Prosimo, izbrišite "+t+" znak";return t!=1&&(n+=t==2?"a":"ov"),n},inputTooShort:function(e){var t=e.minimum-e.input.length,n="Prosimo, vnesite še "+t+" ali več znakov";return n},loadingMore:function(){return"Nalaganje več rezultatov…"},maximumSelected:function(e){var emx=e.maximum, t="Izberete lahko samo "+emx+" postavk";return emx<3&&(t+=emx==2?"i":"o"),t},noResults:function(){return"Brez rezultatov"},searching:function(){return"Iščem…"}}}),{define:e.define,require:e.require}})();
```

Comment:
Please [do not update the dist/ files](https://github.com/select2/select2/blob/master/CONTRIBUTING.md#submitting-a-pull-request), add a file here instead: https://github.com/select2/select2/tree/master/src/js/select2/i18n